index.css

```diff
@@ -0,0 +1,151 @@
+.body {
+  font-family: "Poppins", sans-serif;
+}
+
+.mobile {
+  position: absolute;
+  width: 375px;
+  height: 718px;
+  color: white;
+  display: flex;
+  flex-direction: column;
+  background-image: url(images/header-shapes.png);
+  background-position-y: 50px;
+}
+
+.navbar {
+  position: absolute;
+  width: 375px;
+  height: 92px;
+  background-color: white;
+}
+
+.toolbar {
+  display: flex;
+  flex-direction: row;
+  top: 44px;
+  align-items: center;
+}
+
+img {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+}
+
+.toggle {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  position: absolute;
+  width: 18px;
+  height: 12px;
+  right: 24px;
+  top: calc(50% - 40px / 2);
+  margin-top: 35px;
+  margin-right: 24px;
+}
+
+.logo {
+  width: 76px;
+  height: 20px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+  margin-top: 58px;
```

Comment:
- [Optional] Your `navbar` is looking great, If you look through you can see you have too mach margin from the top that is coming from your margin for the `logo `text. `navbar` is usually come next to the mobile header, hence only a single padding (something like 10px) will be enough.

### [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)
![Capture2](https://user-images.githubusercontent.com/69077061/192083888-a4043e58-797b-443e-aa24-16c770ff6b96.PNG)

## Your design
![Capture](https://user-images.githubusercontent.com/69077061/192083904-acc5c3c9-ff9e-4b34-847a-97357ffab3b8.PNG)